src/Log/Engine/ArrayLog.php

```diff
@@ -57,6 +57,7 @@ class ArrayLog extends BaseLog
      * @param array $context Additional information about the logged message
      * @return void success of write.
      * @see \Cake\Log\Log::$_levels
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
```

Comment:
These will show up in the API docs. We might want to filter these attributes out when building the API docs if we continue to use them.  Another way to ignore these errors would be in the configuration files.

Reply:
The new api docs should show only a specific list of attributes.